tests/db/test_v202208190_to_v202209160.py

```diff
@@ -0,0 +1,42 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+import os
+from typing import Awaitable, Callable, List
+
+import pytest
+
+
+@pytest.mark.db_restore_dump(os.path.join(os.path.dirname(__file__), "dumps/v202208180.sql"))
+async def test_added_exporter_plugin(
+    migrate_db_from: Callable[[], Awaitable[None]],
+    get_columns_in_db_table: Callable[[str], Awaitable[List[str]]],
+    get_custom_postgresql_types: Callable[[], Awaitable[List[str]]],
+) -> None:
```

Comment:
This fixture is not used by this test case.
```suggestion
    get_columns_in_db_table: Callable[[str], Awaitable[List[str]]],
) -> None:
```